examples/c++/Ar_vSTS.cpp

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
I recommend that you make this example a unit test so that it doesn't get lost in the future.